### ISSUE TITLE
Connectors disappear

### ISSUE THREAD

**Issue by alainassaf** [ryancbutler/UnideskSDK#62]
Hello,

With AppLayering version 22.8.0.1003 and version 0.3.27 of the SDK, I've run across 2 issues:

- My Citrix Hypervisor (8.2CU1) connector will disappear from the ELM after or during an OS rev or application build.
- After finalization the VM that was created by ELM will be present, but the storage will be removed. 

I just wanted to mention this in case anyone else has run into any issues. It could be a lot of factors, but I didn't run into any issues during all my testing, but this second month of using the SDK to automate builds, the 2 issues above have pushed us to go back to manual patching. 

Unfortunately, the disappearing connector seems to mess up any existing builds and we've had to revert to a previous backup because we could not cancel running tasks. We also opened a support case with Citrix to see if we can get the ELM working again without restoring from backup. I'll try and update once we get feedback from Citrix support.

Thanks,
Alain

**Comment by ryancbutler** [comment]
Thanks for the info! Will keep open for now.

**Comment by alainassaf** [comment]
Hello,

I've got some feedback on our Citrix case. We ruled out that it wasn't related to the any recent XenServer changes. Citrix support had to get on the ELM and perform a number of Linux and MySQL commands to "unstuck" existing tasks that could not be canceled. The takeaway is that the combination of using the script and having people using the ELM console at the same time created some conflicts in the database. Moving forward I'm going to use the script only when I have exclusive access to our ELM. We have a team of 6 and during our patch cycle there may be 2 or 3 people creating or updating layers in the ELM. Adding the script to this seems to have caused database issues.

Thanks,
Alain
